api/colors.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
@YeonV I think we have this already in https://github.com/LedFx/ledfx_rewrite/blob/main/color/predefined.go